ui/src/config.js

```diff
@@ -24,6 +24,10 @@ export const config = {
       },
     },
   },
+  CIRCULATION: {
```

Comment:
great, did you check ALL config vars in `defaultConfig.js` in `react-invenio-app-ils`? I am wondering if there are more...

Reply:
I double checked it and accordingly to what we discussed with the librarians these are the changes we need to do